miller-columns.scss

```diff
@@ -195,10 +195,9 @@ $mc-active-item-background: govuk-colour("blue");
     padding-top: 0;
     padding-right: 0;
     padding-bottom: 0;
-    border-top: 0;
-    border-right: 0;
-    border-left: 0;
+    border: 0;
     background: transparent;
+    text-decoration: underline;
```

Comment:
Could you add some explanation for these changes?

Reply:
Sure, I've now added a description to this commit:
> Updating to govuk-frontend requires updating the govuk-back-link class we apply to our button element in a similar way govuk-frontend applies it to an anchor element. More on this change on the original PR in govuk-frontend https://github.com/alphagov/govuk-frontend/pull/1753 – more specifically https://github.com/alphagov/govuk-frontend/pull/1753/files#diff-4c6ef8a7bcb3d85bc2f7e7a0cd1fd50cR33